### ISSUE TITLE
Change Toast text prop type to allow react elements

### ISSUE THREAD

**Issue by mondul** [Cogoport/cogo-toast#25]
After updating I'm getting this warning:
```
Warning: Failed prop type: Invalid prop `text` of type `object` supplied to `Toast`, expected `string`.
    in Toast (created by ToastContainer)
    in ToastContainer
```

Currently I'm showing my errors as a list like:

```
cogoToast.error(<>
    <div>* Error 1</div>
    <div>* Error 2</div>
</>)
```

**Comment by anmolmahatpurkar** [comment]
Fixed and merged in #26 in `v3.2.0`

**Comment by fdemello** [comment]
Are we sure this issue was fixed? 

![image](https://user-images.githubusercontent.com/13694949/70272760-cceec000-1776-11ea-8d60-3bff3e785e18.png)
